Makefile

```diff
@@ -1,9 +1,27 @@
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
```

Comment:
The pipeline fail option is a nice touch, but do you know if this works on MacOS? The comment states

> The .SHELLFLAGS variable was introduced with GNU Make 3.82. It has no effect on the version of make installed on Mac OS X, which is GNU Make 3.81.

I like the functionality provided by `.SHELLFLAGS` but if the behavior differs between platforms we should document it at the very least, if not fully remove it so we get consistent builds.

Reply:
removed